### DISCUSSION TITLE
Documentation on how to Docker-ize FlightPHP application.

### DISCUSSION THREAD

**Discussion by iamkarshe** [mikecao/flight#518] [Ideas]
Small documentation on how to use Docker to deploy FlightPHP application, I will share my Dockerfile for my sample project and use ChatGPT to format some step-by-step guide on how to use Docker with FlightPHP.

Ref. links
- https://hub.docker.com/_/php
- https://medium.com/@miladev95/dockerizing-a-php-application-e3c756670336
- https://semaphoreci.com/community/tutorials/dockerizing-a-php-application

**Comment by n0nag0n** [comment]
That's great! You could make a pull request in here or the flightphp.com project for documentation.

FYI the https://github.com/flightphp/skeleton repo has a `docker-compose.yml` file and a `Vagrantfile` for environment management.
